text/0000-serialization.md

```diff
@@ -0,0 +1,187 @@
+- Start Date: 2015-10-16
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+The goal of this RFC is to formalize and document the various serialization formats used by Refract, as well as explicitly list how to handle ambiguity in refracted metadata and attributes.
+
+# Motivation
+
+1. Currently they are not defined in the specification. There is one RFC for Embedded Refract but it is not visible to people looking at the Refract specification repository.
```

Comment:
I believe we should consider the Full version of Refract the main Refract serialization format, and then move the other versions to their own repositories as their own specs. This would mean we'd move the Compact Refract to its own repo along with the Embedded Refract.

That would mean we'd take your recommendations here for Compact, use the RFC for Embedded, and leave the Full version alone. Thoughts?